mmv1/products/compute/terraform.yaml

```diff
@@ -1503,12 +1503,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "machine_image_basic"
         primary_resource_id: "image"
+        primary_resource_name: "fmt.Sprint(\"image\")"
         vars:
           vm_name: "vm"
           image_name: "image"
       - !ruby/object:Provider::Terraform::Examples
         name: "compute_machine_image_kms"
         primary_resource_id: "image"
+        primary_resource_name: "fmt.Sprint(\"image\")"
```

Comment:
Given what I've learned, these tests (triggered by my latest commit push) are likely to fail due to the downstream having `${random_suffix}` added to the end of the name.